app/assets/stylesheets/jquery-ui/custom-theme/jquery-ui-1.8.13.custom.scss

```diff
@@ -59,26 +59,26 @@
 .ui-widget { font-family: Arial,Helvetica,sans-serif; font-size: 1em; }
 .ui-widget .ui-widget { font-size: 1em; }
 .ui-widget input, .ui-widget select, .ui-widget textarea, .ui-widget button { font-family: Arial,Helvetica,sans-serif; font-size: 1em; }
-.ui-widget-content { border: 1px solid #aaaaaa; background: #ffffff url(images/ui-bg_flat_75_ffffff_40x100.png) 50% 50% repeat-x; color: #222222; }
+.ui-widget-content { border: 1px solid #aaaaaa; background: #ffffff image_path('ui-bg_flat_75_ffffff_40x100.png') 50% 50% repeat-x; color: #222222; }
 .ui-widget-content a { color: #222222; }
-.ui-widget-header { border: 1px solid #dc5541; background: #9c0000 url(images/ui-bg_highlight-soft_75_9c0000_1x100.png) 50% 50% repeat-x; color: #ffffff; font-weight: bold; }
+.ui-widget-header { border: 1px solid #dc5541; background: #9c0000 image_path('ui-bg_highlight-soft_75_9c0000_1x100.png') 50% 50% repeat-x; color: #ffffff; font-weight: bold; }
```

Comment:
Color literals like `#dc5541` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#9c0000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Name of function `image_path` should be written in all lowercase letters with hyphens instead of underscores<br>Properties should be ordered background, border, color, font-weight